src/main/java/nextstep/subway/line/domain/Line.java

```diff
@@ -1,41 +1,59 @@
 package nextstep.subway.line.domain;
 
-import nextstep.subway.common.BaseEntity;
-import nextstep.subway.line.dto.LineRequest;
+import javax.persistence.Column;
+import javax.persistence.Embedded;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
 
-import javax.persistence.*;
+import nextstep.subway.common.BaseEntity;
+import nextstep.subway.section.domain.Section;
+import nextstep.subway.section.domain.Sections;
 
 @Entity
 public class Line extends BaseEntity {
-    @Id
-    @GeneratedValue(strategy = GenerationType.IDENTITY)
-    private Long id;
-    @Column(unique = true)
-    private String name;
-    private String color;
-
-    public Line() {
-    }
-
-    public Line(String name, String color) {
-        this.name = name;
-        this.color = color;
-    }
-
-    public void update(Line line) {
-        this.name = line.getName();
-        this.color = line.getColor();
-    }
-
-    public Long getId() {
-        return id;
-    }
-
-    public String getName() {
-        return name;
-    }
-
-    public String getColor() {
-        return color;
-    }
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	private Long id;
+	@Column(name = "name", unique = true)
+	private String name;
+	@Column(name = "color")
+	private String color;
+
+	@Embedded
+	private Sections sections = new Sections();
+
+	public Line() {
+	}
```

Comment:
```suggestion
	protected Line() {
	}
```

public으로 노출될 필요가 있을까요?

Reply:
protected로 수정했습니다.